packages/app-store/routing-forms/pages/router/[...appPages].tsx

```diff
@@ -47,11 +48,21 @@ export const getServerSideProps = async function getServerSideProps(
   }
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const { form: formId, slug: _slug, pages: _pages, ...fieldsResponses } = queryParsed.data;
-  const form = await prisma.app_RoutingForms_Form.findUnique({
+  const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(context.req.headers.host ?? "");
+
+  const form = await prisma.app_RoutingForms_Form.findFirst({
```

Comment:
Changed to findFirst because you can have user has in where clause for findUnique